src/lib/arrays.js

```diff
@@ -0,0 +1,109 @@
+const getNthElement = (index, array) => {
+  if (index >= array.length) {
+    return array[index - array.length];
+  }
+  return array[index];
+};
+
+// const getNthElement = (index, array) => {
```

Comment:
All the commented out unused versions of the functions here should be removed, they are just cluttering up your working code